### PR TITLE
Improve travis.yml

### PR DESCRIPTION
**Pull request by mischah**
Add node 4 to travis.yml and make use of the container-based-infrastructure to start up faster.

See https://docs.travis-ci.com/user/workers/container-based-infrastructure/ for details.

### CODE REVIEW

File: app/templates/travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
+sudo: false
```
Comment (sindresorhus) [inline app/templates/travis.yml]: This is not needed anymore.

File: app/templates/travis.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 language: node_js
 node_js:
   - 'stable'
```
Comment (sindresorhus) [inline app/templates/travis.yml]: can be replaced with `'5'`

File: app/templates/travis.yml (hunk 1 above)
Comment (mischah) [inline app/templates/travis.yml]: oha. good to know :kissing_heart:

**Comment by mischah** [comment]
Did a force push to update the PR.

Now its just about adding node 4 and renaming `stable` with `5` :relaxed:

**Pull request merged**